torch/nn/qat/modules/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .linear import Linear
```

Comment:
You can `# @lint-ignore-every PYTHON3COMPATIMPORTS` here to avoid linter erroring out

Reply:
does this only work for this file?